package/MDAnalysis/core/topology.py

```diff
@@ -13,9 +13,39 @@
 # J. Comput. Chem. 32 (2011), 2319--2327, doi:10.1002/jcc.21787
 #
 
-"""
-Topology object --- :mod:`MDAnalysis.core.topology'
-===================================================================
+"""\
+Core Topology object --- :mod:`MDAnalysis.core.topology`
+========================================================
+
+.. versionadded:: 0.16.0
+
+:class:`Topology` is the core object that holds all topology information.
+
+TODO: Add in-depth discussion.
```

Comment:
Can you make an issue for that. So we are sure to remove all TODO:s in the docs before the release